core/tauri/scripts/core.js

```diff
@@ -133,7 +133,7 @@
 
   // drag region
   document.addEventListener('mousedown', (e) => {
-    if (e.target.hasAttribute('data-tauri-drag-region') && e.buttons === 1) {
+    if (e.composedPath().find(e => e instanceof Element && e.hasAttribute('data-tauri-drag-region'))  && e.buttons === 1) {
```

Comment:
this would have the same issue described here https://github.com/tauri-apps/tauri/pull/3031#issuecomment-986776732